### PR TITLE
Changeling Tweak Bonanza: Electric Boogaloo

### PR DESCRIPTION
**Pull request by Crystalwarrior**
Soft-reverts https://github.com/HippieStationCode/HippieStation13/pull/1922 (only reverts the death sting and coma sting changes, armblade change is unaffected)

Fixes https://github.com/HippieStationCode/HippieStation13/issues/2255

This PR also changes reccomended enemies for both Traitors and Changelings to 2, that should decrease the amount of both on lowpop rounds.
- rscadd: "Headslugs are now an ability changelings start with. You can become a headslug if you're MMI'd or decapitated."
- rscdel: "Removed comatose sting (2 FULL MINUTES OF UTTER AND COMPLETE HELPNESSNESS IS NOT OKAY), reverted death sting back to the original (wasting toxin instead of venom, tier 3)"
- tweak: "Tweaks changeling headslug timing to be more accurate"
- tweak: "Reduced recommended_enemies on both Traitor and Changeling gamemodes to 2 so they're more lowpop-friendly"
- tweak: "Tweaked changeling regeneration to use revive() proc"
- tweak: "Increased headslug speed to be the same as humans so you're not a complete joke."
- tweak: "Tweaked absorbtions to not work on NPC humans (monkey-humans/mind datum-less humans)"
- tweak: "Para-sting no longer alerts the victim of being stung until they're stunned"
- tweak: "Better chameleon skin animations, chameleon skin now takes less time to turn fully invisible (approx. 3 seconds)"
- tweak: "Reduced point cost for syanptic enhancement"
- tweak: "Removed death sting, added purge sting for chemical purging"
- tweak: "Shorter ventcrawl delay for horror lings and headslugs"
- tweak: "Made gib requirement for devouring for horror form be 200 instead of 300 damage"
- tweak: "REDUCED STING RANGE BACK TO IT'S ORIGINAL, NO LONGER CAN YOU STING SOMEONE ONE TILE AWAY FROM YOU JESUS FUCK"
- tweak: "Tweaked changeling objectives:"
- tweak: "   Genomes goal decreased so you have to absorb from 2 to 4 genomes."
- tweak: "   Tweaked assasinate and maroon objectives to give you changeling's codename if they become your target."
- rscdel: "   Removed \"escape with identity\" objective."
- bugfix: "Fixed husked people not showing up husked"
- bugfix: "Fixed being able to handcuff people with missing arms"

### CODE REVIEW

**Comment by Crystalwarrior** [comment]
If we want to change amount of changelings per players, those with server config access should change changeling_scaling_coeff from 6 to 12 or something!

**Comment by Sarcalogo** [comment]
> Removed comatose sting (2 FULL MINUTES OF UTTER AND COMPLETE HELPNESSNESS IS NOT OKAY), reverted death sting back to the original (wasting toxin instead of venom, tier 3

Get good.

**Comment by Kierany9** [comment]
>implying it takes skill to middleclick

**Comment by ghost** [comment]
I only have 1 finger so it takes me a lot of effort to click someone.

**Comment by Sarcalogo** [comment]
Changelings without those stings may as well be at the mercy of whatever shit head power gamer strolls by with an anti stun. Parasting takes like 15 seconds to activate, alerts the person they were stung, and stuns you for as long as like one stun baton hit. Cryo sting does like 5 damage and slows a person down, that's it. Old deathsting was ridiculously easy to cheese away, the minute you realized you took toxin damage you had about two minutes to rush to medbay and swallow one anti toxin pill or one calomel syringe.

I say get good, because really you need to get good. Changelings in the past had the exact same sting comatose sting is now except they could get it at round start, nobody complained for years. This was with RANGED stings too which let you extend your sting by 3 tiles. 

Do you not want to die to changelings? Don't try to fight tier 4 changelings alone. Don't try to fight them with a fucking stun baton. I realize it's easy to cheese the genomes, but I'm fixing it so you cannot absorb monkey men.

**Comment by Crystalwarrior** [comment]
Actually parasting no longer alerts the victim. Forgot to add that to the changelog.

**Comment by Kierany9** [comment]
>Do you not want to die to changelings? Don't try to fight tier 4 changelings alone.

You mean the guys who can be disguised as a minimum of nine people at any given moment? Yeah, I mean it's so easy to identify them, and trying to fight them is my choice which I always pick when unarmed.

**Comment by Sarcalogo** [comment]
By the time there is a tier 4 ling on station there are now at least 8 husks on station, most likely more because of the other changelings. You are well aware there are changelings on station at this point, and the crowd is thinned. Don't stand next to people you don't trust, run away from people who try to stand next to you for no reason. Changelings are supposed to inspire paranoia, they are supposed to be hard to kill, turns out you may die if there are changelings on station. If you can't deal with that, why are you playing the game?  

By the way, at that point they could just flash and silence sting you and the result would still be the same. Or arm blade your head off. Or you know they could fire a syringe at you from a range that gives the exact same effect and then kill you.

 Again, the Comatose and Death Sting are mainly meant to be powergamer shut downs.

**Comment by Kierany9** [comment]
>By the time there is a tier 4 ling on station there are now no more monkeys in genetics

Fixed that for you

**Comment by Sarcalogo** [comment]
Why don't you go ahead and read the above, monkey men no longer give any genomes. It makes no sense to remove Comatose and Death sting given that.

**Comment by Crystalwarrior** [comment]
Comatose and Death sting are both a little too bullshit in their core design, so it still makes perfect sense to remove them.
Heck there's a reason why Tier 4's Horror Form is not invincible and is pretty slow, even if it can dish out a huge amount of damage it's still not an insta-win button in any way, shape or form. I feel that all of the abilities should pose some risk-reward and challenge regardless of tier (in this case every tier ramps up both the risk and reward most of the time, having a smooth "stealth" to "loud" curve)

**Comment by Crystalwarrior** [comment]
Not to mention, paralysis sting was turned into a stealth-sting so you get a stun in without the victim realising it before it's too late. I'd say it's a decent buff and makes comatose sting even more unneccesary.

I honestly think that if we're going to add more changeling abilities they should be something a little more interesting than "more 1-click-win stings", but instead actual abilities.

**Comment by Kierany9** [comment]
>Why don't you go ahead and read the above

It also removes coma sting and makes death sting less stupidly uncurable, and stupidly really is the right word here. Tier-4 changeling is not the wizard and is not particularly that hard to get considering how stupidly durable they are(and considering how many there are).

>By the way, at that point they could just flash and silence sting you and the result would still be the same. Or arm blade your head off. Or you know they could fire a syringe at you from a range that gives the exact same effect and then kill you.

The difference between these and death sting is that you need to cick more than once and maybe use your brain a little before and during clicking.

**Comment by Sarcalogo** [comment]
>Tier-4 changeling is not the wizard

They are supposed to be overpowered, actually. But even so, they are easily beatable if you approach them correctly. 

 >Is not particularly that hard to get considering how stupidly durable they are

Beg to differ, again, it now requires you to have killed 8 people, possibly more if you have not saved up your points, that is a lot of room to mess up. There is no cheesing with monkey men anymore. 

>The difference between these and death sting is that you need to cick more than once and maybe use your brain a little before and during clicking.

You need to have used your brain to survive long enough as a changeling to absorb 8 people. You also act like it takes much skill or effort to walk up to someone pull a stun prod, or just plain shoot them (you'll likely have a gun at this point), silence sting, cuff, and then absorb them. Comatose sting just makes it so you don't have to cuff the person and sting them if you are quick. Very little functional difference between the two, you still end up dead. 

You still do not understand the purpose of the comatose sting. Changelings prior to this were absolutely trivialized by anti stun chemicals and incredibly commonly used power gaming gear. It would be very difficult to win as a changeling if someone like Cappy, Agent Speer, or even Sir John was playing security. What comatose sting does, is that it discourages the shitty power gaming lone wolf mindset by actually making changelings a credible threat.

**Comment by Crystalwarrior** [comment]
I can at least see where you're coming from, Sarcalogo, and honestly I do think Changelings need a few more cool abilities at tiers 3-4.
But I do NOT think that having stings that put the person you used it on on a full 2-minute uncounterable stun with a mute toxin to boot. In no way is it fair or even balanced, no matter what tier you put it in.
Not to mention comatose sting was a straight up buff of paralysis sting, meaning that if you got coma sting you had zero reason to come back to para-sting anyway. This isn't good design.

**Comment by Kierany9** [comment]
>They are supposed to be overpowered, actually. But even so, they are easily beatable if you approach them correctly. 

No, they aren't. And no they aren't. Surprisingly Horror form is easier to beat than a tier four in my experience.

>You also act like it takes much skill or effort to walk up to someone pull a stun prod, or just plain shoot them

Which is why I say it's not particularly hard to get eight absorbs.

>What comatose sting does, is that it discourages the shitty power gaming lone wolf mindset by actually making changelings a credible threat.

>Changelings prior to this were absolutely trivialized by anti stun chemicals and incredibly commonly used power gaming gear.

This is actually a valid point but IMO it's a problem with combat as a whole and its incredible reliance on stuns, not something to be fixed with a "lol fuck you gg no re game over dipshit" sting.

**Comment by Sarcalogo** [comment]
>Not to mention, paralysis sting was turned into a stealth-sting so you get a stun in without the victim realising it before it's too late.

Except against people who use anti stuns. Then it just outs you as a changeling and gets you lasered to death and incinerated. And even so, it would always make more sense to silence sting. After what is it, 30 seconds have passed, the person you stung is now most likely far away from you and can now call out that you are a changeling. It also does NOTHING to stop someone who is attacking you, preventing your death, which is one of the utilities parasting was supposed to have that I had to reinvoke in the comatose sting. With the ridiculous beheading nerf included as well (it's not as if its hard to kill a headslug, even if you are faster now, you're KOS by everybody anyway) you are shit on by anyone with a stun baton or two.

>No, they aren't.

Yes they are, changelings are supposed to be a threat. A tier 4 changeling is a changeling in peak form. It should be difficult to kill.

> And no they aren't

They are. All you have to do is approach them with more than one person and a few energy guns  If you shoot them, they can either use an adrenaline sac to get up or get lasered to death. If they adrenaline up they are now operating on a MAX of 70 chemicals which means one more stun and they can no longer use comatose sting. At that point you can safely melee them while the other person fires lasers. Once they are dead, it's pretty much over. Again, you're not supposed to be able to 1 v1 a changeling, especially at that level, it's just not going to happen.

>Surprisingly Horror form is easier to beat than a tier four in my experience.

Of course it is, because whoever designed Horror Form horribly underpowered it. It's a waste of genomes even prior to comatose sting existing.

>Which is why I say it's not particularly hard to get eight absorbs.

But there is a difference between stunning, cuffing, and absorbing one person and surviving long enough to do it to 8 (again possibly more) people. Each time you absorb someone, you are putting yourself at a greater risk You have to dispose of the corpse, get rid of the evidence (which takes time and is dangerous) or simply stash the corpse in a locker or something and hope for the best. Each time you kill someone or are in the process of killing someone, the chances of you getting caught raise. It's much harder to get 8 absorbs than it is to kill a person.

**Comment by TheOneAndOnlyCreeperJoe** [comment]
You claim that cryosting counters lone wolfs.
Then why the hell does it cost 50, effectively allowing you to use it twice if you take the 1-cost upgrade that increases your capacity by 25?

**Comment by Sarcalogo** [comment]
I dunno, I wanted to have a scaling increase. Death sting is clearly better than comatose sting so it cost more points and cost more chemicals. It's ideally supposed to work after one anti-stun. I suppose you could increase the cost to 75.

**Comment by blobbernaut** [comment]
bugfix: "Fixed being able to handcuff people with missing arms"

wait what

it's going to become a valid strategy to chop off one of your arms.

**Comment by carlarc** [comment]
Fuck you Sarcalogo. Having a instakill at one click away, no matter how many people you have absorbed, is utter bullshit and Jesus i can't believe your shit got merged.

This all seems fucking fantastic. Merge ASAP

**Comment by myontasm** [comment]
Imagine being so dense that you think coma sting was a good idea

**Comment by Sarcalogo** [comment]
Imagine being so dense that you cannot formulate a proper argument to paragraphs of text and can only insult people in a passive aggressive way. Imagine being so new, you don't even know that comatose sting is literally just old para sting which had been in the game for years and is still in the game on some servers. Imagine being so shit at the game that you cry to remove anything that takes the slightest bit of smarts to avoid.  

Oh wait, there's no need to imagine that, we have such a faggot right here and his name is Limed00d.

**Comment by blobbernaut** [comment]
what might be fine on a roleplay server where murderboning is very discouraged is different from what's fine on this server, sarcalogo. Nobody checks crew monitors here. 

Although I fucking hate literally every single mechanic changelings have except for their ability to regenerate, I think they're a pile of stinking fucking garbage far worse than blob mode.

**Comment by TDcoolguy300** [comment]
Code looks fine.

**Comment by ghost** [comment]
Indifferent to headslugs being roundstart available (You're never gonna see a borged ling though considering they tend to go straight into the cremater /gibber.)
Yes to comatose sting removal
Yes to death sting not doing 12.8 damage on the first tick before decaying into histamine.
No to death sting being utterly countered by dexalin (Cryo sting is already cucked by coffee) (Maybe make it deal more toxin damage when you lack blood? Prolonged encounters would get you killed then.)
Yes to making headslugs world.time based rather than ticks.
Yes to recommended enemy changes. (Shouldn't mulligan take care of it if there's less antags?)
Yes to lings being less snowflakey.
Not sold on headslugs being as fast as humans (Considering they're a passive last resort now, right? They should be better than nothing.)
Yes to ending monkey farming.
Does the message telling you're parasting really matter? I mean, it occurs just before the effects and not on sting. Lemme know if I'm missing something on that.
Double yes to genome reduction (Although it might be a bit too easy on high pop, I guess we could add more genome scaling later?)
Yes to the changeling target objective change.
Yes to removal of objectives that may or may not be possible.

File: code/game/gamemodes/changeling/powers/absorb.dm [hunk 1]
```diff
@@ -8,22 +8,25 @@
...
 	var/mob/living/carbon/target = G.affecting
 	if((target.disabilities & NOCLONE) || (target.disabilities & HUSK))
 		user << "<span class='warning'>The DNA of [target] is ruined beyond usability!</span>"
-		return
+		return 0
 	if(!ishuman(target))
 		user << "<span class='warning'>[target] is not compatible with this ability.</span>"
-		return
+		return 0
+	if(!target.mind)
```
Comment (JohnGinnane) [inline code/game/gamemodes/changeling/powers/absorb.dm:27]: Do disconnected humans have a mind value?

**Comment by JohnGinnane** [comment]
Code looks good, but crystal I think we should do a private lobby to test things like cuffs, head slugs and other changes

File: code/game/gamemodes/changeling/powers/absorb.dm (hunk 1 above)
Comment (Crystalwarrior) [inline code/game/gamemodes/changeling/powers/absorb.dm:27]: Yeah

**Comment by Crystalwarrior** [comment]
Handcuffs, etc. are tested on my test server, I always test my changes on one.

I might buff death sting later.

**Comment by ghost** [comment]
Don't get me wrong, as is, this is a better alternative to current death sting but it'd be nice for it to not get cucked by an even more common place chem than coffee.

**Comment by blobbernaut** [comment]
MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY MERGE THIS IMMEDIATELY

**Comment by Gandalf2k15** [comment]
This is one of the most needed PR's there is. Please merge.

**Comment by carlarc** [comment]
Change deathsting to 15 chloral maybe?

**Comment by Kierany9** [comment]
>Change deathsting to 15 chloral maybe?

Do you know how piss weak chloral is?

**Comment by carlarc** [comment]
Better than venom! Also no I don't.

**Comment by Kierany9** [comment]
15u of chloral will do a grand total of zero damage. Chloral needs to spend 50 ticks in its your body before it starts doing any damage.

Then its damage skyrockets and before you know it you've died in your sleep.

**Comment by carlarc** [comment]
So it sucks anyways.

**Comment by carlarc** [comment]
Just fucking remove deathsting, it's either useless or OP

**Comment by Sarcalogo** [comment]
If you are really keen on removing the current death sting change it to Amantin. You've still ignored all logic in the face of feels though. 

Enjoy getting killed over and over and over again as a changeling now, as powergaming retards loaded up on meth, HoS armor, telescopic armor, and a shit ton of guns effortlessly shut you down by lasering you to death, dragging you into a small room, pulling out their powergaming beheading tool, beheading you, and then killing your headslug as you are trapped in a 4x4 room with no vents and doors you can't open. ENJOOOOOOOOOOY.

**Comment by Sarcalogo** [comment]
And if you are really keen on comatose sting, just cut the amount of time it stuns you down by half, rename it to something like "upgraded parasting", and remove the mute toxin. It's still enough to shut down power gamers with meth, but now they can call out for help at least, and have a stronger chance of survival. 

Those are my compromises.

**Comment by myontasm** [comment]
ok

**Comment by Crystalwarrior** [comment]
I like the death sting compromise, but "upgraded parasting" is pretty silly imo.
I wonder, though. What if changelings could spit gear-melting acid at tier 3?

**Comment by Kierany9** [comment]
>Change death sting to inject Amanitin

This is actually good.

(why dont we just add a meth sting that makes powergamers overdose and die because meth/bath sants now work like amatin but with a bit less damage and more stun)

**Comment by Crystalwarrior** [comment]
How about Purge Sting? Purges chems in target's body.

~~maybe comes with Purge Self?~~

**Comment by Sarcalogo** [comment]
>maybe comes with Purge Self?

Do not do this, or power gaming lings will become unstoppable.

**Comment by Crystalwarrior** [comment]
Honk.

**Comment by ghost** [comment]
What do you mean purge self? That's already an ability. Panacea or whatever, removes harmful chems and stuff.
Edit: Disregard that I'm wrong.

**Comment by Crystalwarrior** [comment]
Oh yeahhhhhhh, anatomic panacea. It removes diseases but I had no idea it also removed dangerous chems.

**Comment by Sarcalogo** [comment]
Panacea does not remove chemicals and for a good reason. If changelings can flush chemicals, a shitty HoS powergaming ling becomes unbeatable. Terrible idea, and by the way maybe I shouldn't be telling you this, because it will just be abused, but you can purge chemicals as a ling by Lesser Forming. Hackole does this the few times he is fucked over as a ling and continues to murderbone.

**Comment by Crystalwarrior** [comment]
Hmm, iirc lesser form doesn't even deal any brute damage to the ling does it? tbh I think it should deal you ~20 damage so if you spam-abuse it you have to either have a medkit handy or some fleshmend, which means more chemicals spent = more openings for the crew to exploit.

**Comment by ghost** [comment]
To be honest, lesser form puts you into a state where you drop your gear and shite. Also considering it makes you a monkey, it should be elods of genetic damage to stop spamming it. Otherwise reagents should transfer (probably won't work 100% intended for some chems that rely on cycles or some shit.)

**Comment by Crystalwarrior** [comment]
Genetic damage is an utter BITCH to work with, though. Almost nothing heals it short of cryo.

Unless anatomic panacea heals genetic damage it's kind of a no. I'd stick to brute :V

**Comment by Kierany9** [comment]
>And a purge sting is absolutely useless because it won't purge chemicals that aren't in the body yet (dental implants)

Not if it injected Calomel(like stings should instead of just changing vars looking at you hallucination sting) which would deal tox damage pretty quickly while also removing other chems from the body. However, it can't kill the target, it just leaves them at 20 HP.

**Comment by blobbernaut** [comment]
SARC IF THE HOS HAS ALL THAT AND DOES ALL THE PLANNING THEY -SHOULD- WIN.
GOOD FOR THE HOS FOR KILLING THE SHITTIEST BULLSHIT NO COUNTERPLAY ANTAG.

Why is this so hard to understand for you? Why do you refuse to acknowledge
that the HOS loaded up on meth with a bunch of lasers SHOULD kill a ling
1v1 if they have no armor or tasers? Or that regular crew members, or hell,
traitors too, should have a chance in hell to kill one? What you don't get
is that lings suck. They're a terrible, unhealthy antag that has one
playstyle: MURDERBONE! A ling dying is GOOD for the round.

Lings should have defense and utility tools almost exclusively. Their
weapon should be their utility, their robustness and their loot. NOT.
STINGS. Stings should be defensive or reactive, or utility. Blind sting
does it right, it's extremely well balanced and can win you fights without
just murdering them in three seconds. They can still fight back. They
should be a fantasy of some extremely difficult to kill, regenerating
monster, that kills with cunning. They should be able fake implants too,
for example.

**Comment by Gandalf2k15** [comment]
Yep the HEAD OF SECURITY should not be an easy target. Especially a power gaming one. Seriously I have been fucked over soo many times as HoS and traitor to fucking bullshit stings that require noooo skill. Fuck lings and fuck their op stings. Delete these stings please..

**Comment by Gandalf2k15** [comment]
I just killed 21 people with the deathsting, including all of ERT. One click and 5 seconds dead. Don't nerf pls.

**Comment by Crystalwarrior** [comment]
Removing death sting.

But!!! Changelings will get purge sting instead (which injects calomel).

**Comment by Crystalwarrior** [comment]
UPDATED CHANGELOG: Removed death sting, tweaked and added some shit.

**Comment by blobbernaut** [comment]
how bout a stasis sting that makes the target knocked out for 30 seconds, but prevents interacting with them in any way except pulling them, i.e. can't be damaged

**Comment by Crystalwarrior** [comment]
Stupid and retarded. No thanks.

We should really stray away from "stun lol" stings, changeling can do that pretty easily with stun prods and other stuff.

**Comment by blobbernaut** [comment]
not as stupid and retarded as bloody death sting

**Comment by Crystalwarrior** [comment]
Hence why it got removed :^)

**Comment by calibraptor** [comment]
+1

Death & Comatose sting are kinda bullshit.

**Comment by Sarcalogo** [comment]
You really don't get that sting is absolutely useless do you? Tell me, when would a changeling EVER use purge sting, give some examples.

**Comment by Gandalf2k15** [comment]
You dont get that the death sting is far too op to be in the game do you?

**Comment by dreamcrab** [comment]
i'm not sold on the purge sting, what else can replace the death sting? death sting is awful.

the rest of it seems alright. how jacked up are debrain objectives? those are always fun

**Comment by Sarcalogo** [comment]
I already said that you could replace wasting toxin with amanitin, which would kill you in about 3 minutes instead of 6 seconds, while still being able to be countered if you realized you got stung (scanning yourself or using a sleeper).

**Comment by ghost** [comment]
Then a maxed out changeling would just run up and down the station stealth stinging people in hallways. Changelings should certainly be fearsome at level 4 but they should be fearsome in the same way that horror form is. Horror form is at least very strong, but you can have a semblance of fun taking it on and it's fun for the ling because it's fukken horror form.

Death sting as is and Comatose sting as is are bad design simply because they're 1 click win items that can't be used against them and for the most part, are 100% stealthy.

**Comment by ghost** [comment]
Although it's certainly important to find some sort of alternative to these stings or else changelings are just gonna always go horror form.

**Comment by Crystalwarrior** [comment]
In my humble opinion I think having ANY stings that kill your victim directly shouldn't be in the game.
Heck, cryosting sure comes close to bein a kill-sting, but coffee is an easy counter for it, for one, and for two it's more useful for slowing your victim down to a crawl and catching them offguard or as a finisher rather than a straight up kill sting.

You have to keep in mind that STINGS ARE STEALTHY. MOST OF STINGS DON'T EVEN PROVIDE A WARNING TO THE VICTIM. IF YOU WANT TO HAVE DEATH STING USE A FUCKING ARMBLADE.

**Comment by Kierany9** [comment]
>or else changelings are just gonna always go horror form.

IMO this isn't a bad thing

**Comment by ghost** [comment]
Outright removing the current stings is just gonna mean that every ling is going to full horror form every day all day. That's going to lead to stagnated gameplay.

We need more different upgrades that can be considered.

Why not something like the changeling makes a volatile pilot-able clone of their current identity (non-ling) that will gib after a few minutes? At least there's some sebelence of interest for all parties involved fighting a hiding ling.

Consider EMAG for example. It's incredibly versatile and is almost certain someone will buy it during a round. But, they will be short of some other telecrystals for other things. You're not REQUIRED to pick up an emag since it has an impact on the rest of your kit and anything an emag does, a Captain's ID does better. It's a clutch item but it's useful to everyone. So not everyone buys it because they'd rather have 6 TC and spend it in other things that are harder to emulate. So traitor rounds won't stagnate to the exact same build every game.

If changelings just had horror form, it would just be the same rush for horror form every day for the rest of changeling life.

**Comment by Kierany9** [comment]
Ling definitely does need some new powers that aren't stings.

I think all these problems are symptoms of a greater inherent problem in the design of the gamemode but nobody really has any ideas on how to solve it.

**Comment by Crystalwarrior** [comment]
Tbh we need more things like chameleon skin; passive abilities that make lings not only stealthy, but also creepy as fuck.

**Comment by JohnGinnane** [comment]
Idea: Ability which changes your scream to a shriek that can disarm nearby
people, with chance of disarm depending on distance to ling as well as
whether they can in sight of ling

**Comment by JohnGinnane** [comment]
Note: Might not make people fall over or freeze, just disarm them!

**Comment by Crystalwarrior** [comment]
that's just the two shrieks avaliable at tier 3 tho.

**Comment by carlarc** [comment]
Some guy suggested adding back meat shield and also some sort of throwable toxic bone spears
i think it sounds really interesting and probably fun to fight against

**Comment by Crystalwarrior** [comment]
I'll do it in a separate PR tbh, this one is already bloated enough

**Comment by Crystalwarrior** [comment]
Pretty sure this got two head approvals, though maybe someone else would like to weigh in?

**Comment by TDcoolguy300** [comment]
All i can say is: Fuck deathsting

**Comment by Crystalwarrior** [comment]
:+1:  MY MAN

**Comment by calibraptor** [comment]
Merge this already

**Comment by TDcoolguy300** [comment]
Stab + Limed00d approved, merging.

**Pull request merged**